developers/blobstream.md

```diff
@@ -2,7 +2,7 @@
 description: Learn how to integrate your L2 with Blobstream
 ---
 
-# Integrate with Blobstream
+# Blobstream: Streaming modular DA to Ethereum
```

Comment:
Should this say "Blobstream X" for consistency or just Blobstream?

Reply:
Just wanted to double-check

Reply:
https://github.com/celestiaorg/docs/pull/1379#discussion_r1477709788